stdlib/LinearAlgebra/test/symmetric.jl

```diff
@@ -88,6 +88,8 @@ end
             @testset "Unary minus for Symmetric/Hermitian matrices" begin
                 @test (-Symmetric(asym))::typeof(Symmetric(asym)) == -asym
                 @test (-Hermitian(aherm))::typeof(Hermitian(aherm)) == -aherm
+                @test (-Symmetric(Symmetric(ones(Bool, 1, 1)))) == Symmetric(-ones(Bool, 1, 1))
```

Comment:
Why the double wrapping? Maybe something like
```
@test (-Symmetric(Bool[1 1; 0 0]))::Symmetric{Int,Matrix{Int}} == [-1 -1; -1 0]
@test (-Hermitian(Bool[1 1; 0 0]))::Hermitian{Int,Matrix{Int}} == [-1 -1; -1 0]
```
would be a more interesting test-case?

```suggestion
                @test (-Symmetric(Bool[1 1; 0 0]))::Symmetric{Int,Matrix{Int}} == [-1 -1; -1 0]
```